### ISSUE TITLE
Expose AnyView object to widget.render()

### ISSUE THREAD

**Issue by Aldlevine** [manzt/anywidget#193]
[This solution](https://github.com/manzt/anywidget/issues/28#issuecomment-1399711348) is no longer possible since 0.5.0 because the raw `AnyView` object is no longer exposed to the `render()` callback (it's now being filtered by [`extract_context`](https://github.com/manzt/anywidget/blob/90464667249404fcf0db743ddb53b1308eeb1eef/packages/anywidget/src/widget.js#L109-L125)).

Because of this, I see no good way to build a container widget (my current effort). I would like to use `AnyWidget` to bootstrap my project, but without access to the lower level `ipywidgets` stuff, I don't think it's feasible.

I can happily submit a PR to expose the `AnyView` object. My local version is exposing it as another property on the result of `extract_context`: e.g. `return { model, el: view.el, view};`

**Comment by manzt** [comment]
Thanks for the issue and for the suggestion!

I opened #194 to expose `model.widget_manager`. If you look into the prior solution and the internals of `create_child_view`, you'll see that they are just aliases for using this lower-level object. 

I rolled out the changes in 0.5.0 to narrow anywidget's API surface area, which will allow us to more flexibility (compared to ipywidgets) to change things without breaking things for widget developers.

Woud you mind giving #194 a review?

**Comment by Aldlevine** [comment]
Thanks for the quick response! 

This looks great. I totally agree with minimizing the public API. I didn't realize that `AnyView.create_child_view` was just an alias to `IWidgetManager.create_view`.

I tested this out in a local jupyter notebook and it works very well. I'm able to pass child widgets from python and access their models/views from within my widget in JS. I'm also able to recursively nest to an arbitrary depth, and all widgets I've tested work seamlessly. As far as I can tell, this should support everything I need.
